.changeset/tricky-singers-brush.md

```diff
@@ -0,0 +1,6 @@
+---
+'@envelop/core': patch
```

Comment:
I think this can be removed? no need for a patch if not actual code was changed ;)

Reply:
Right, no need for a patch. I wasn't sure how to write the changeset so left as is, but have removed the path info.

Thanks.